test/built-ins/Temporal/PlainDateTime/prototype/round/roundingmode-halfexpand-is-default.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindatetime.prototype.round
+description: Half-expand is the default rounding mode
+features: [Temporal]
+includes: [temporalHelpers.js]
+---*/
+
+const dt = new Temporal.PlainDateTime(1976, 11, 18, 14, 23, 30, 123, 456, 789);
+
+const units = ["day", "hour", "minute", "second", "millisecond", "microsecond", "nanosecond"];
+
+units.forEach((unit) => {
+  TemporalHelpers.assertPlainDateTimesEqual(
+    dt.round({ roundingmode: "halfExpand", smallestUnit: unit }),
+    dt.round({ smallestUnit: unit }),
```

Comment:
I would prefer you to copy the actual expected values from `test/built-ins/Temporal/PlainDateTime/prototype/round/roundingmode-halfexpand-basic.js` rather than calling `dt.round()` twice per test

Reply:
Sure, done.